addons/event/models/event_event.py

```diff
@@ -224,10 +224,19 @@ def _lang_get(self):
     lang = fields.Selection(_lang_get, string='Language',
         help="All the communication emails sent to attendees will be translated in this language.")
     # ticket reports
+    badge_format = fields.Selection(
+        string='Badge Dimension',
+        selection=[
+            ('A4_foldable', 'A4 foldable'),
+            ('A6', 'A6'),
+            ('four_per_sheet', '4 per sheet')
```

Comment:
Seems it is 'A4_french_fold', according to "The Internet'

Reply:
Relooking at various website, seems the *french fold* is 4 / A4 (fold in 4 a bit like we do generally, not like a promotional folder or stuff like that) and *half fold* is 2 / A4 .

Note that we could rename *french fold* to *french fold :wine_glass: * .